modules/basics/src/main/java/com/opengamma/strata/basics/currency/Money.java

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.basics.currency;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+
+import org.joda.beans.JodaBeanUtils;
+import org.joda.convert.ToString;
+
+import com.google.common.collect.ComparisonChain;
+import com.google.common.math.DoubleMath;
+
+/**
+ * An amount of a currency, rounded to match the currency specifications.
+ * <p>
+ * This class is similar to {@link CurrencyAmount}, but only exposes the rounded amounts.
+ * The rounding is done using {@link BigDecimal}, as BigDecimal.ROUND_HALF_EVEN. Given this operation,
+ * it should be assumed that the numbers are an approximation, and not an exact figure.
+ * <p>
+ * This class is immutable and thread-safe.
+ */
+public class Money implements FxConvertible<Money>, Comparable<Money>, Serializable {
```

Comment:
This class isn't serialisable using Joda Beans. It should define a `parse` method with a `@FromString` annotation.